plugin/sampling/strategystore/static/options.go

```diff
@@ -22,9 +22,11 @@ import (
 )
 
 const (
-	// SamplingStrategiesFile contains the name of CLI opions for config file.
+	// SamplingStrategiesFile contains the name of CLI options for config file.
 	SamplingStrategiesFile           = "sampling.strategies-file"
 	samplingStrategiesReloadInterval = "sampling.strategies-reload-interval"
+	// SamplingStrategiesURL contains the name of CLI option for config file URL.
+	SamplingStrategiesURL = "sampling.strategies-url"
```

Comment:
I think the intention of the ticket was to allow `sampling.strategies-file` to contain a URL. A new flag seems redundant.